### ISSUE TITLE
TestFlight

### ISSUE THREAD

**Issue by Pikachupikachup** [emuPlace/Limon#12]
I used to be very active on your original project, emuThreeDS[1], and I was very, how do you say, intrigued in the fact that you would bring this to TestFlight. When you canceled that idea I could understand, but now that you have made a new project **I wanted to ask of you would bring _this_ to TestFlight.**

[1]I don't have a phone so because of the new changes(I posted a few codes on git, rlly just basic things)I have to use the app GitHub but just never bothered to download it

**Comment by jarrodnorwell** [comment]
Because Apple will very likely ban the account that was used to upload the app to TestFlight, it's very unlikely I'll be publishing it there.

I'd love to as it'd allow everyone to install it so much easier but I'd rather not risk my account on it (see EggNS)

**Comment by Pikachupikachup** [comment]
Yeah, that was about the same answer that came when I the original's TestFlight was cancelled. I understand, just wondering.

**Comment by Pikachupikachup** [comment]
Wait, I couldn't find where they stated that they even TRIED to put Egg NS on TestFlight. Could you link and quote where the post, tweet, website page, etc. is?

**Comment by jarrodnorwell** [comment]
Only picture I was able to find of the TestFlight screen

https://www.facebook.com/share/p/mHQ8iYt8adt2vhxe/?mibextid=WC7FNe

**Comment by Pikachupikachup** [comment]
Got it, thank so much for looking
